department-of-veterans-affairs/caseflow
# Refactored searchable dropdown to use appropriate aria roles

Resolves [CASEFLOW-1012](https://vajira.max.gov/browse/CASEFLOW-1012)

Part 1 of the [508 compliance stack](https://github.com/department-of-veterans-affairs/caseflow/pull/16374)

### Description
Resolves the issue where JAWS was reading the word "Edit" after reading the label for a searchable dropdown

### Acceptance Criteria
- [ ] JAWS no longer reads the word "edit" after focusing the Docket Type searchable dropdown in the add hearing day form

### Testing Plan
1. First setup your environment to use JAWS following [these steps](https://github.com/department-of-veterans-affairs/appeals-team/blob/master/Engineering/accessibility-jaws-setup.md)
2. Once you are at the screen to select a user, choose `BVASYELLOW` and navigate to the hearings schedule: http://10.0.2.2:3000/hearings/schedule
3. Click the "Add Hearing Day" link
4. Click on the Docket Type label so that JAWS will read the label and ensure that it does not announce the word "edit" after announcing the label
